Source/DafnyCore/Rewriters/InductionRewriter.cs

```diff
@@ -253,7 +289,10 @@ List<List<Expression>> ComputeInductionTriggers(List<Expression> inductionVariab
     var triggersCollector = new Triggers.TriggersCollector(finder.exprsInOldContext, Reporter.Options, moduleDefinition);
     var quantifierCollection = new Triggers.ComprehensionTriggerGenerator(quantifier, Enumerable.Repeat(quantifier, 1), Reporter);
     quantifierCollection.ComputeTriggers(triggersCollector);
-    var triggers = quantifierCollection.GetTriggers();
+    // Get the computed triggers, but only ask for those that do not require additional bound variables. (An alternative to this
+    // design would be to add {:matchinglooprewrite false} to "quantifier" above. However, that would cause certain matching loops
+    // to be ignored, so it is safer to not include triggers that require additional bound variables.)
+    var triggers = quantifierCollection.GetTriggers(false);
```

Comment:
From the comment above I don't immediately see why we don't need triggers for those additional bound variables.

Reply:
The idea was that if there are any `_t#0` rewrites, then we don't want to do automatic induction.